source/developers/how-tos/maintain-a-repo.rst

```diff
@@ -111,6 +111,35 @@ Update your readme until it meets the above specification.
 
 .. _readme specification: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0003-readme-specification.html
 
+Your CCs are Aligned With Your Expectations as Maintainer
+*********************************************************
+
+A repo that you take on may already
+have other CCs that have merge rights on it.  You should review which CCs have
+write access to your repo on the `CCs Wiki Page`_.  You should communicate with
+your CCs and ensure that you are aligned with them on the architectural goals
+of the repo, especially if there are particular kinds of changes that you as
+the maintainer would like to have more input on.
+
+.. warning::
+
+   If the repo in question is an MFE, devs might have access to it via the
+   `frontend-all` group.  Be sure to check for that.
```

Comment:
I don't know if this is entirely sufficient, I could see relatively soon adding groups for data repos and perhaps more.

Reply:
Can we list the set of groups at the top of the CC wiki paged so that people can be aware of them.  Then this could be a more generic message to look out for groups?

Reply:
Oh wait, I didn't see you suggesting the same thing above.  Cool, I'll update the wiki page since we had the same idea 😄